packages/flutter/lib/src/painting/unicode.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Constants for useful Unicode characters.
+///
+/// Currently, these characters are all related to bidirectional text.
+///
+/// See also:
+///
+///  * <http://unicode.org/reports/tr9/>, which describes the Unicode
+///    bidirectional text algorithm.
+class Unicode {
```

Comment:
Should be in foundation, rather than painting, no?

Reply:
Yeah that's probably reasonable.